help/de/Help_ch_GUIMenus.xml

```diff
@@ -462,13 +474,11 @@
                 </entry>
 
                 <entry nameend="c3" namest="c2">
-                  <para>Speichert die aktuell geöffnete Datei unter einem anderen Namen.
-                  </para>
-
-                  <warning>
-                    <para>Speichern Sie Ihre Datendatei NICHT in <filename class="directory">~/.gnucash/books</filename>.
+                  <para>Speichert die aktuell geöffnete Datei unter einem anderen Namen.<footnote>
+                    <para>Bis zu &app; Version 3.0 sollten Sie aufpassen, dass Sie Ihre Datendatei NICHT in <filename class="directory">~/.gnucash/books</filename> speichern.
```

Comment:
Vor 3.0/bis 2.6.x

Reply:
ENTITY vers-last-2
habe ich mal definiert in Anlehnung an ähnliche Variablen in Website und im Wiki.